terraform/environments/xhibit-portal/network-infrastructure.tf

```diff
@@ -448,6 +499,27 @@ resource "aws_security_group_rule" "portal-http-to-waf-lb" {
   source_security_group_id = aws_security_group.waf_lb.id
 }
 
+resource "aws_security_group_rule" "portal-http-from-prtg-lb" {
+  depends_on               = [aws_security_group.prtg_lb, aws_security_group.portal_server]
+  security_group_id        = aws_security_group.portal_server.id
+  type                     = "ingress"
+  description              = "allow all traffic from DB"
```

Comment:
Can you correct these descriptions? This is allowing access to the load balancer from the portal server

Reply:
description       = "allow web traffic to get to prtg Load Balancer over SSL "